### PR TITLE
Update BTXpress

### PR DESCRIPTION
**Pull request by natalia2493**
- Removed IMDB Description as it is not needed
- Removed Audio Category

### CODE REVIEW

File: src/Jackett.Common/Definitions/btxpress.yml [hunk 1]
```diff
@@ -12,7 +12,6 @@
     categorymappings:
       - {id: 1, cat: Movies, desc: "Movies"}
       - {id: 2, cat: TV, desc: "TV"}
-      - {id: 3, cat: Audio, desc: "Music"}
```
Comment (HDVinnie) [inline src/Jackett.Common/Definitions/btxpress.yml]: Just curious why remove this when they still offer the Music category on site?

**Comment by flightlevel** [comment]
@natalia2493 as per @HDVinnie any reason for removing audio?

**Comment by natalia2493** [comment]
We are planning on taking out the Audio category completely.

**Comment by flightlevel** [comment]
merged, thanks @natalia2493

**Pull request merged**